### PR TITLE
Removing Coverletters and old resume list

### PR DESCRIPTION
**Pull request by usbsnowcrash**
## Feature Branch PR

### CODE REVIEW

File: lib/cb/config.rb [hunk 1]
```diff
@@ -57,17 +57,11 @@ def set_default_api_uris
...
-      @uri_application_registered ||= '/v3/application/registered'
       @uri_application_external ||= '/v1/application/external'
       @uri_application_form ||= '/cbapi/job/:did/applicationform'
       @uri_company_find ||= '/Employer/CompanyDetails'
       @uri_countries ||= '/consumer/datalist/countries'
       @uri_cover_letters ||= '/consumer/coverletters'
-      @uri_cover_letter_list ||= '/v1/coverletter/list'
-      @uri_cover_letter_retrieve ||= '/coverletter/retrieve'
-      @uri_cover_letter_update ||= '/coverletter/edit'
-      @uri_cover_letter_delete ||= '/coverletter/delete'
```
Comment (oochelz) [inline lib/cb/config.rb]: Need to remove the resume route :)

**Comment by ausmith** [comment]
I'm not seeing anything glaring, going to let Chelsea finish her run through of the code as well.

**Comment by oochelz** [comment]
Just remove the resume list route and I'm good.

File: lib/cb/config.rb (hunk 1 above)
Comment (usbsnowcrash) [inline lib/cb/config.rb]: done

**Pull request merged**